packages/@aws-cdk/aws-scheduler-alpha/lib/schedule.ts

```diff
@@ -59,6 +59,47 @@ export interface ScheduleTargetProps {
   readonly retryAttempts?: number;
 }
 
+/**
+ * A time window during which EventBridge Scheduler invokes the schedule.
+ */
+export class FlexibleTimeWindowMode {
+  /**
+   * FlexibleTimeWindow is disabled.
+   */
+  public static off(): FlexibleTimeWindowMode {
+    return new FlexibleTimeWindowMode('OFF');
+  }
+
+  /**
+   * FlexibleTimeWindow is enabled.
+   */
+  public static flexible(maximumWindowInMinutes: Duration): FlexibleTimeWindowMode {
+    if (maximumWindowInMinutes.toMinutes() < 1 || maximumWindowInMinutes.toMinutes() > 1440) {
+      throw new Error(`maximumWindowInMinutes must be between 1 and 1440, got ${maximumWindowInMinutes.toMinutes()}`);
+    }
+    return new FlexibleTimeWindowMode('FLEXIBLE', maximumWindowInMinutes);
+  }
+
+  /**
+   * Determines whether the schedule is invoked within a flexible time window.
+   */
+  public readonly mode: string;
+
+  /**
+   * The maximum time window during which the schedule can be invoked.
+   *
+   * Must be between 1 to 1440 minutes.
+   *
+   * @default - Required if mode is FLEXIBLE.
```

Comment:
This isn't the correct use of the `@default` tag. Should describe the default if mode is not flexible